### ISSUE TITLE
Project Identifier: 0845793f0872eccfa8b55bc8c14fae79

### ISSUE THREAD

**Issue by sirmaestro512** [aws-amplify/amplify-cli#11102]
after I update amplify cli seems im stuck on amplify push or pull and says env dev not identified now

**Comment by sirmaestro512** [comment]
⠹ Fetching updates to backend environment: dev from the cloud.⠋ Building resourc
⠇ Building resource api/rntv✅ GraphQL schema compiled successfully.

Edit your schema at /Users/kennethboothjr/builds/RNTV/amplify/backend/api/rntv/schema.graphql or place .graphql files in a directory at /Users/kennethboothjr/builds/RNTV/amplify/backend/api/rntv/schema
✔ Successfully pulled backend environment dev from the cloud.
⠋ Copying video resources. This may take a few minutes...✖ There was an error initializing your environment.
🛑 Cannot read property 'dev' of undefined

Resolution: Please report this issue at https://github.com/aws-amplify/amplify-cli/issues and include the project identifier from: 'amplify diagnose --send-report'
Learn more at: https://docs.amplify.aws/cli/project/troubleshooting/

⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠸ Copying video resources. This may take a few minute
⠼ Copying video resources. This may take a few minute
⠴ Copying video resources. This may take a few minute
⠦ Copying video resources. This may take a few minute
⠧ Copying video resources. This may take a few minute
⠇ Copying video resources. This may take a few minute
⠏ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minute
⠙ Copying video resources. This may take a few minute
⠹ Copying video resources. This may take a few minute
⠋ Copying video resources. This may take a few minutes...

**Comment by josefaidt** [comment]
Hey @sirmaestro512 :wave: thanks for raising this! Would you mind porting this over to our [bug template](https://github.com/aws-amplify/amplify-cli/issues/new?assignees=&labels=&template=1.bug_report.yaml)? By doing so this will help us better understand the context of the issue

**Comment by tmaidment** [comment]
Having the exact same issue with the Video API, stuck on Copying Video Resources, only after the latest API upgrade. Is there any way to downgrade for the time being?

**Comment by josefaidt** [comment]
Hey @tmaidment this is a community plugin, which I encountered a few other issues with during my reproduction. Would you mind filing an issue on the amplify-video repo? https://github.com/awslabs/amplify-video

**Comment by tmaidment** [comment]
Hi @josefaidt, I do not believe it is an issue with amplify-video. I downgraded to 10.0.0, and everything worked as is. @sirmaestro512, maybe try downgrading to 10.0.0 to see if that fixes your problem.

**Comment by sirmaestro512** [comment]
So yea I’m the future they should say wait to upgrade just in as it breaks
your app.  In fact upgrade to new cli will do this and/or changed your
project root directory thus causing an improper installation. My fox just
now to still keep the updated cli and my project was to reinstall aws cli
via the install prompt where I respecified where to install and for whom.
After doing this I am now able to reconnect to the backend and either pull
in my current site or start fresh. I’m personally doing a fresh backend
only cause I wasn’t that far into the tables being filled in and last night
before I did all this I made some changes in my cloud formation template
that I thought would fix the issue so now I’m not sure if those changes
were needed. However if you didn’t make that mistake like me in making
changes to your site or cloudformation template or etc then just pull in
your backend (if u didn’t delete the backend folder) and good to go.

**Comment by sirmaestro512** [comment]
I meant my fix

**Comment by sirmaestro512** [comment]
ok update after doing all the above once you do amplify add video again you get the same issue...the issue is with amplify add video plugin and appears for some reason not be be fully compatible with the new aws cli.  I am now downgrading my cli and will keep everyone posted.  Any help from amplify video plugin community would be awesome

**Comment by josefaidt** [comment]
Hey @sirmaestro512 and @tmaidment let's use this issue to track the behavior with the third party video plugin https://github.com/aws-amplify/amplify-cli/issues/11112 

I'm able to reproduce all of the issues described here and in the linked issue, and will work with the Amplify CLI team in more depth to investigate this

**Comment by gabriellacerda** [comment]
This bug is a bit scary, here at @dalmataworks is happening the same thing to our core team, downgrading the cli to v10.0.0 solved the issue, thanks to @tmaidment

**Comment by sirmaestro512** [comment]
The bug is very scary however, like everyone else, once I downgraded back to v10.0.0 amplify video add works again.  Seems, once the amplify new version is updated as release on GitHub maybe then it will be safe to upgrade the cli, but not now.  Also I opened a ticket on the plugin page and still no response.  My concern is that that plugin that so many of us utilize is no longer being maintained.  Seems like there needs to be a workaround or alt option for those of us who still need the benefits of that plugin or similar setup integrating with our amplify cloud formation stacks of our existing projects.

**Comment by sirmaestro512** [comment]
I’m seriously debating only because I’m not too far into it to just redo my
backend with the updated cli and call it a day. I see a lot of us are
having this issue.  I been went as far as changing settings and etc in
cloud formation so on that alone I think I’m a just redo my backend real
quick. Better now with the minimum.
